tests/test_utils.py

```diff
@@ -677,7 +677,7 @@ def test_check_configured(self, mock_config, expected_result):
             assert extraction_service._check_configured() is expected_result
 
     @pytest.mark.asyncio
-    async def test_extract_text(self, mock_responses):
+    async def test_extract_text(self, mock_responses, patch_logger):
```

Comment:
I don't feel this is worth testing

Reply:
I normally wouldn't either, but for this project we had a strict requirement of filenams being trackable in logs. I just wanted to ensure that the expected filename appears in the debug log.